app/src/main/java/com/iic/shopingo/ui/request_flow/views/CreateRequestItemListView.java

```diff
@@ -108,4 +123,8 @@ public void onItemEdited(View view, String value) {
     }
   }
 
+  public interface OnRequestItemListChanged {
+    public void onItemListChanged(View view);
```

Comment:
Yeah, but from what I saw, it's kind of a convention. Can I just go ahead and leave it out?